### DISCUSSION TITLE
Guide to setup IntelliJ import order (to satisfy checkstyle)

### DISCUSSION THREAD

**Discussion by Brutus5000** [yahoo/elide#2170] [Show and tell]
The default import order of IntelliJ will screw up in this project. If you want to use auto-import you will have to make changes in your IntelliJ settings:

Preferences -> Editor -> CodeStyle -> Java -> Tab imports

Make sure to only set it to "Project" scheme. You will probably not want this for other projects. At the bottom your "Import Layout" should look like this:

![image](https://user-images.githubusercontent.com/6843462/122619674-461be280-d091-11eb-976e-d1dc12d08c6b.png)

**Comment by aklish** [comment]
👍
